COPYRIGHT

```diff
@@ -0,0 +1,19 @@
+Except as otherwise noted (below and/or in individual files), this project is
+licensed under the Apache License, Version 2.0 <LICENSE-APACHE> or
```

Comment:
there seems to be an `or` too much here.

Reply:
You're right -- me culpa!